vue/test/components/ripe-pickers/ripe-pickers.test.js

```diff
@@ -1,19 +1,158 @@
 const assert = require("assert");
 const base = require("../../base");
 
-const ripe = require("ripe-sdk");
+const ripe = require("ripe-sdk").ripe;
 
-describe("RipePickers", () => {
-    it("should instantiate the component", () => {
-        const ripeInstance = new ripe.Ripe("dummy", "cube", {
-            version: 52
-        });
-        const component = base.getComponent("RipePickers", {
-            props: {
-                ripe: ripeInstance
+class MockRipeSdk {

```

Comment:
Try to mimic the structure of https://github.com/ripe-tech/ripe-white/blob/master/vue/test/components/base-personalization-plugin/base-personalization-plugin.test.js#L8 so that we have a "base" mock with some commonly needed stuff.